dmtestutils/login.py

```diff
@@ -0,0 +1,39 @@
+from flask import Blueprint
+from dmutils.user import User
+from flask_login import login_user
+
+login_for_tests = Blueprint('login_for_tests', __name__)
+
+
+USERS = {
+    '123': {
+        'id': 123,
+        'supplier_name': 'Supplier Name',
+        'name': 'Name',
+        'emailAddress': 'email@email.com',
+        'role': 'supplier',
+        'supplierId': 1234,
+    },
+    '234': {
+        'id': 234,
+        'name': 'Buyer',
+        'emailAddress': 'buyer@email.com',
+        'role': 'buyer',
+    },
+}
+
+
+@login_for_tests.route('/auto-login')
+def auto_login():
```

Comment:
Is it worth putting a brief docstring explaining where these routes are likely to be called -
 `BaseApplicationTest.login()`, right? (I guess `BaseApplicationTest` itself should live in this repo eventually...)

Reply:
Correct, but I might just put in PR's for all of the repo's as that won't take very long (famous last words).